packages/mui-material/src/Divider/Divider.d.ts

```diff
@@ -30,7 +30,7 @@ export interface DividerOwnProps {
   /**
    * If `true`, the divider will have a lighter color.
    * @default false
-   * @deprecated Use <Divider sx={{ bgcolor: '#eee' }} /> (or any color) instead.
+   * @deprecated Use <Divider sx={{ bgcolor: '#eee' }} /> (or any color) instead. [How to migrate](/material-ui/migration/migrating-from-deprecated-apis/)
```

Comment:
Now that we added the codemod I would do:

```suggestion
   * @deprecated Use <Divider sx={{ opacity: 0.6 }} /> (or any opacity or color) instead. [How to migrate](/material-ui/migration/migrating-from-deprecated-apis/)
```